meerkat/datapanel.py

```diff
@@ -817,3 +820,29 @@ def _copy_data(self) -> object:
 
     def __finalize__(self, *args, **kwargs):
         return self
+
+    def mean(self) -> DataPanel:
+        columns = self.columns
+        d = {}
+
+        for column in columns:
+            try:
+                from meerkat.columns.tensor_column import TensorColumn
+                mean = 0
+                if isinstance(self.data[column], TensorColumn):
+                    print("Tensor Column!", column)
+                    tensor = self.data[column].to_tensor()
+
+                    print("Got tensor", tensor)
+                    mean = tensor.double().mean().item()
+                    print("got mean!")
+                else:
+                    mean = self.data[column].mean()
+                d[column] = mean
+            except Exception as e:
```

Comment:
Why the catch all exception? If there are specific issues we're expecting let's have try except blocks around more targetted code with specific exception types

Reply:
There is not really a very specific issue here.

Reply:
If there isn't, let's not except as it could lead to silent bugs